spec/models/task_spec.rb

```diff
@@ -23,30 +24,40 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  context 'factory tests' do
-    subject { build(:task) }
-    it { is_expected.to be_valid }
-  end
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user_id: user.id) }
+  let!(:project) { create(:project, workspace_id: workspace.id) }
+  let!(:task1) { create(:task, project_id: project.id) }
```

Comment:
fyi: `create(:project, workspace: workspace)` to get rid of extra queries on validations